src/arrayMethodSort.js

```diff
@@ -5,8 +5,37 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    const compare = compareFunction || compareAsStrings;
```

Comment:
You can just set a default value in function params

Reply:
Oh, stupid me :)